go/libraries/doltcore/sqle/dolt_index.go

```diff
@@ -159,6 +160,21 @@ func (di *doltIndex) ID() string {
 	return di.id
 }
 
+// IsUnique implements sql.Index
+func (di *doltIndex) IsUnique() bool {
+	return di.unique
+}
+
+// Comment implements sql.Index
+func (di *doltIndex) Comment() string {
+	return "" // TODO: store comments
```

Comment:
In `go/libraries/doltcore/sqle/tables::GetIndexes`, where we create the `doltIndex` objects, you have access to the index. Just like you're doing for `Unique` you can do for for `Comment` too.

Reply:
Oh snap, I didn't think we had this! Added.